packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -1222,4 +1256,100 @@ describe('TokensController', () => {
       ).toStrictEqual([]);
     });
   });
+  describe('updateTokensName', () => {
+    const tokenList = [
+      {
+        address: '0x123',
+        name: '',
+        symbol: 'ABC',
+        aggregators: [],
+        decimals: 0,
+        iconUrl: '',
+        occurrences: 1,
+      },
+      {
+        address: '0x456',
+        name: '',
+        symbol: 'DEF',
+        aggregators: [],
+        decimals: 0,
+        iconUrl: '',
+        occurrences: 1,
+      },
+      {
+        address: '0x789',
+        name: '',
+        symbol: 'GHI',
+        aggregators: [],
+        decimals: 0,
+        iconUrl: '',
+        occurrences: 1,
+      },
+    ];
+
+    it('update token name when a matching token is found in the tokenList', async () => {
```

Comment:
By using the word "when" in this test, you seem to be referencing some kind of logic. So, what happens when a matching token is _not_ found in the tokenList? In other words, do you need a test that exercises the opposite case?

Reply:
Added an opposite test case! Amazing catch!